src/stores/Research/research.store.tsx

```diff
@@ -280,8 +294,32 @@ export class ResearchStore extends ModuleStore {
     }
   }
 
-  public deleteResearchItem(id: string) {
-    this.db.collection('research').doc(id).delete()
+  @action
+  public async deleteResearch(id: string) {
+    try {
+      const dbRef = this.db.collection<IResearchDB>(COLLECTION_NAME).doc(id)
+      const researchData = await toJS(dbRef.get('server'))
+
+      const user = this.activeUser
+
+      if (id && researchData && user) {
+        await this._updateResearchItem(dbRef, {
+          ...researchData,
+          _deleted: true,
+        })
+
+        if (this.activeResearchItem !== undefined) {
+          this.allResearchItems = this.allResearchItems.filter(
+            (researchItem) => {
+              return researchItem._id !== researchData._id
+            },
+          )
+        }
+      }
```

Comment:
Think this can be simplified.

```suggestion
      const user = this.activeUser

      if (id && user) {
        await dbRef.update({_id: id, _deleted: true})
        }

        if (this.activeResearchItem !== undefined) {
          this.allResearchItems = this.allResearchItems.filter(
            (researchItem) => {
              return researchItem._id !== id
            },
          )
        }
      }
```